### PR TITLE
Expand intro to deserialization

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready.

File: docs/standard/serialization/system-text-json-how-to.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ You can [implement custom converters](system-text-json-converters-how-to.md) to
 
 ## How to read JSON as .NET objects (deserialize)
 
-To deserialize from a string or a file, call the <xref:System.Text.Json.JsonSerializer.Deserialize%2A?displayProperty=nameWithType> method.
+A common way to deserialize JSON is to first create a class with properties that represent one or more of the JSON attributes. Then, to deserialize from a string or a file, call the <xref:System.Text.Json.JsonSerializer.Deserialize%2A?displayProperty=nameWithType> method. You can deserialize either synchronously or asynchronously. Any JSON attributes that aren't represented in your class are ignored.
```
Comment (tdykstra) [inline docs/standard/serialization/system-text-json-how-to.md:172]: Strictly speaking, JSON doesn't have attributes

```suggestion
A common way to deserialize JSON is to first create a class with properties and fields that represent one or more of the JSON properties. Then, to deserialize from a string or a file, call the <xref:System.Text.Json.JsonSerializer.Deserialize%2A?displayProperty=nameWithType> method. You can deserialize either synchronously or asynchronously. Any JSON properties that aren't represented in your class are ignored.
```

**Review by tdykstra — APPROVED** [review]
LGTM with a slight change in terms.

File: docs/standard/serialization/system-text-json-how-to.md [hunk 2]
```diff
@@ -169,7 +169,7 @@ You can [implement custom converters](system-text-json-converters-how-to.md) to
 
 ## How to read JSON as .NET objects (deserialize)
 
-To deserialize from a string or a file, call the <xref:System.Text.Json.JsonSerializer.Deserialize%2A?displayProperty=nameWithType> method.
+A common way to deserialize JSON is to first create a class with properties and fields that represent one or more of the JSON properties. Then, to deserialize from a string or a file, call the <xref:System.Text.Json.JsonSerializer.Deserialize%2A?displayProperty=nameWithType> method. You can deserialize either synchronously or asynchronously. Any JSON properties that aren't represented in your class are ignored.
```
Comment (gewarren) [inline docs/standard/serialization/system-text-json-how-to.md:172]: @tdykstra Is this accurate? I think it adds some nice extra info about how the class is incorporated into the process.

```suggestion
A common way to deserialize JSON is to first create a class with properties and fields that represent one or more of the JSON properties. Then, to deserialize from a string or a file, call the <xref:System.Text.Json.JsonSerializer.Deserialize%2A?displayProperty=nameWithType> method. For the generic overloads, you pass the type of the class you created as the generic type parameter. For the non-generic overloads, you pass the type of the class you created as a method parameter. You can deserialize either synchronously or asynchronously. Any JSON properties that aren't represented in your class are ignored.
```

Comment (tdykstra) [reply docs/standard/serialization/system-text-json-how-to.md:172]: Sounds good.

**Pull request merged**